dist/mpulse.js

```diff
@@ -1505,6 +1509,13 @@ code.google.com/p/crypto-js/wiki/License
                 return;
             }
 
+            if (!configJson.beacon_url || configJson.rate_limited) {
+                warn("config.json shows rate limiting, no beacons will be sent!");
```

Comment:
@nicjansma are there other reasons besides rate-limiting why `beacon_url` might be missing?  If so, you may want to change the comment text to avoid misleading people.

Reply:
Today it's only if `rate_limited` exists